stardog/test/postinstall_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"testing"
+
+	batchv1 "k8s.io/api/batch/v1"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/stretchr/testify/assert"
+)
+
+var tplPostInstallJob = []string{"templates/post-install-job.yaml"}
+
+func Test_PostInstallJob_Should_Not_Exist(t *testing.T) {
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"stardog.adminPassword":     adminPassword,
+			"replicaCount":	"0",
+		},
+	}
+
+	output, err := helm.RenderTemplateE(t, options, helmChartPath, releaseName, tplPostInstallJob)
+
+	assert.NotEmpty(t, err)
```

Comment:
The `assert.NoError(t, err)` is better suited for testing errors. please use that.

Reply:
It's the exact opposite: `assert.Error(t, err)`

Reply:
riiight, mixed that up :)